crates/bevy_sprite/src/mesh2d/mesh2d_vertex_output.wgsl

```diff
@@ -1,11 +1,14 @@
 #define_import_path bevy_sprite::mesh2d_vertex_output
 
-@location(0) world_position: vec4<f32>,
-@location(1) world_normal: vec3<f32>,
-@location(2) uv: vec2<f32>,
-#ifdef VERTEX_TANGENTS
-@location(3) world_tangent: vec4<f32>,
-#endif
-#ifdef VERTEX_COLORS
-@location(4) color: vec4<f32>,
-#endif
+struct MeshVertexOutput {
+    @builtin(position) clip_position: vec4<f32>,
```

Comment:
Same comment here about fragment stage input builtin position not being clip space, so this naming could be confusing.